articles/storage/blobs/storage-dotnet-how-to-use-blobs.md

```diff
@@ -20,385 +20,370 @@ ms.author: tamram
 
 [!INCLUDE [storage-selector-blob-include](../../../includes/storage-selector-blob-include.md)]
 
-[!INCLUDE [storage-check-out-samples-dotnet](../../../includes/storage-check-out-samples-dotnet.md)]
+This tutorial will show you how to use [Azure Blob Storage](https://docs.microsoft.com/azure/storage/blobs/storage-blobs-introduction) with ASP.NET and Visual Studio 2017. It will show how to upload, list, download, and delete blobs in the context of a simple Web API project that works with images.
```

Comment:
This is ASP.NET Core? Correct?

Reply:
This is ASP.NET for now. I'll look into changing this to ASP.NET.

Reply:
I see, i think this is problematic as the photo shows ASP.NET Core and then we can't really run this on mac or linux at all. Seems like we have an opportunity here to really create a cross-platform guide with this :)